src/diy/smol-slimes/smol-hardware.md

```diff
@@ -0,0 +1,210 @@
+
+ - BMI270
+ - ICM-42688-P
+ - ICM-42688-V
+ - ICM-45686
+ - ISM330BX
+ - ISM330DHCX
+ - ISM330DLC
+ - LSM6DS3
+ - LSM6DS3TR-C
+ - LSM6DSL
+ - LSM6DSM
+ - LSM6DSO
+ - LSM6DSR
+ - LSM6DSV
+ - LSM6DSV16B
+
+### 🧲 Magnetometers
+ - AK09940
+ - <div class="tooltip-text-container">BMM150 ⚠️
+    <span class="tooltip-text">Sensor driver has not been tested.</span>
+   </div>
+ - <div class="tooltip-text-container">BMM350 ⚠️
+    <span class="tooltip-text">Sensor driver has not been tested.</span>
+   </div>
+ - IIS2MDC
+ - IST8306
+ - IST8308
+ - LIS2MDL
+ - <div class="tooltip-text-container">LIS3MDL ⚠️
+    <span class="tooltip-text">Sensor driver has not been tested.</span>
+   </div>
+ - MMC5983MA
+
+### 🟩 Sensor Modules with IMU and Magnetometer
+```admonish warning
+ Please note that the most common sensor modules are not supported.
+```
+
+Why combined?
+- Reduces size.
+- Simplifies integration.
+- Ensures consistent alignment of all sensors.
+
+##### Meia IMU + Magnetometer Modules
+Meia produces and sells IMU's with an onboard magnetometer. These have form factor suitable for stacked builds. Meia, is a member of the SlimeVR Discord.
+
+<table>
+    <tr>
+        <th>IMU + Magnetometer</th>
+        <th>Product Page</th>
+    </tr>
+    <tr>
+        <td><a href="../diy/imu-comparison.md#ICM-45686">ICM-45686</a> + IST8306</td>
+        <td><a href="https://store.kouno.xyz/products/icm-45686-ist8306-module">store.kouno.xyz</a></td>
+    </tr>
+    <tr>
+        <td>LSM6DSR + IST8306</td>
+        <td><a href="https://store.kouno.xyz/products/lsm6dsr-ist8306-module">store.kouno.xyz</a></td>
+    </tr>
+</table>
+
+### 🖲️ Buttons
+Push buttons and momentary switches are utilized to control the tracker. The functions of this button—Reset, Calibration, Pairing, Deep Sleep, and entering DFU Mode—depend on the number of press combinations. A tracker can be equipped with either a reset button, a user-specified (SW0) button, or both.
+
+The reset button is designed to support all functionalities. If an user-specified button (SW0) is defined, it will be utilized instead.
+
+If a button is unavailable, tweezers can be used to short the pins for the initial tracker setup.
+
+### 🕹️ Switches
+A slide switch can be used to physically disconnect a battery. Some boards have a high standby power draw and will require a switch.
+
+If a switch is not utilized, a tracker can enter Deep Sleep mode by pressing and holding down the user-specified button (SW0).
+
+### 🔋 Batteries
+Most boards support a 3.7V Li-ion or LiPo battery. Batteries are rated by their capacity (usually in mAh), and they also have a maximum safe charging rate—often expressed as “C.” Charging at 1C means you’re using a current equal to the battery’s capacity. For example, a 1000mAh battery charged at 1A (which is 1C) should reach full charge in about one hour. However, charging at this maximum rate can stress the battery.
```

Comment:
charging time does not correlate directly to charging rate, better to remove a guess

Reply:
also, this is a pretty long explanation of batteries, try to be more concise